### DISCUSSION TITLE
Why not SVG?

### DISCUSSION THREAD

**Discussion by ghost** [KaTeX/KaTeX#3629] [Q&A]
Why is HTML favored over SVG for rendering?
Is it possible to render to SVG instead with KaTeX?

**Comment by edemaine** [comment]
See #375. I think the short answer is that SVG output is difficult because we'd need to compute the exact horizontal placement of everything, whereas currently we just do that for vertical placement. HTML takes care of a lot of the horizontal layout for us.
